File: DirectoryListing/app/src/main/java/com/example/directorylisting/api/WebServiceDetails.java
/*
 * Copyright © 2017 Michael Steele.
 * All rights reserved.
 */

package com.example.directorylisting.api;

import okhttp3.OkHttpClient;

public class WebServiceDetails {
    public WebServiceInterface service = null;
    public OkHttpClient client = null;
    public okhttp3.Call.Factory factory = null;
    public int cacheSize = 0;
}

File: DirectoryListing/app/src/main/java/com/example/directorylisting/api/WebServiceDetails.kt
/*
 * Copyright © 2017 Michael Steele.
 * All rights reserved.
 */

package com.example.directorylisting.api

import okhttp3.OkHttpClient

class WebServiceDetails {
    var service: WebServiceInterface? = null
    var client: OkHttpClient? = null
    var factory: okhttp3.Call.Factory? = null
    var cacheSize = 0
}
